applications/FemToDemApplication/custom_processes/expand_wet_nodes_process.cpp

```diff
@@ -46,6 +46,11 @@ void ExpandWetNodesProcess::Execute()
             }
         }
     }
+    for (auto it_elem = mrModelPart.Elements().ptr_begin(); it_elem != mrModelPart.Elements().ptr_end(); ++it_elem) {
+        (*it_elem)->SetValue(PRESSURE_EXPANDED, false);
```

Comment:
```suggestion
        it_elem->SetValue(PRESSURE_EXPANDED, false);
```

Reply:
it does not compile if I remove the *... I suppose that the iterator has not the SetValue

Reply:
It is supposed to have it. This is very strange

Reply:
Can you try to use the loop as: ?¿
~~~cpp
#pragma omp parallel for
        for(int i=0; i<static_cast<int>(rModelPart.Elements().size()); i++) {
            auto it_elem = rModelPart.ElementsBegin() + i;
            it_elem->SetValue(PRESSURE_EXPANDED, false);
        }
~~~

Reply:
better use variable utils for this

Reply:
it compiles if the it_elem is inside an omp for!! :)